src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/Example1.txt

```diff
@@ -13,6 +13,15 @@ public class Test {
      */
     private void methodWithValidCommentStyle() {}
 
+    // ok below, @return tag automatically inserts a period after the text
+    /**
+     * {@return something}
+     */
+    private String methodWithValidCommentStyle2()
```

Comment:
```
{@return {@code true} if this object has been initialized, {@code false} otherwise}
```

And 
`private boolean isInitialized(){ return true; }`

Goal is to show user good example of usage, as nuance with this tag was surprise even for me. I didn't not know how to use it in this way.

Reply:
excuse me if I didn't get your point, do you want to add this example to the documentation?

Reply:
@romani all changes were done in the last commit, please can you finish the review?

Reply:
@romani Ping